books/free-programming-books-subjects.md

```diff
@@ -363,10 +363,10 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Bayesian Reasoning and Machine Learning](http://web4.cs.ucl.ac.uk/staff/D.Barber/pmwiki/pmwiki.php?n=Brml.HomePage)
 * [Deep Learning](http://www.deeplearningbook.org) - Ian Goodfellow, Yoshua Bengio, Aaron Courville
 * [Deep Learning for Coders with Fastai and PyTorch](https://github.com/fastai/fastbook) - Jeremy Howard, Sylvain Gugger (Jupyter Notebooks)
-* [Deep Learning with PyTorch](https://pytorch.org/assets/deep-learning/Deep-Learning-with-PyTorch.pdf) - Eli Stevens, Luca Antiga, Thomas Viehmann (PDF)
+* [Deep Learning with PyTorch](https://livebook.manning.com/book/deep-learning-with-pytorch/) - Eli Stevens, Luca Antiga, Thomas Viehmann
```

Comment:
only free for a minute or so. just remove it

Reply:
@eshellman I saw it's available at The Internet Archive when the site was cached. Should we keep that copy? Or should I remove it?

Reply:
remove it